storage101.md

```diff
@@ -867,24 +789,24 @@ template: admin
 # PersistentVolume: Dynamic provisioning
 
 .center[
-  <img src="provisioning6.png" width="50%"/><br/>
+  <img src="provisioning8.png" width="50%"/><br/>
 ]
 
+
 ---
 
 template: admin
 
-# PersistentVolume: Dynamic provisioning
+# PersistentVolume: Manual provisioning
 
-.center[
-  <img src="provisioning7.png" width="50%"/><br/>
-]
+- "Brownfield" use case.
+  - Using data of old apps.
```

Comment:
kind of a sparse slide? Maybe a quick image of what pre-provisioning "looks like"?

Reply:
If I add a picture I shall describe it and I thought we're going to defer pre-provisioned PVs as too advanced.

I added the binding back, just in two pictures and I will try to be very quick when describing it